apps/condo/domains/billing/schema/RegisterBillingReceiptFileService.test.js

```diff
@@ -20,6 +20,7 @@ describe('RegisterBillingReceiptFileService', () => {
     beforeAll(async () => {
         billingTestUtils = new BillingTestUtils()
         await billingTestUtils.init()
+        process.env.ADDRESS_SERVICE_CLIENT_MODE = 'fake'
```

Comment:
Seems really strange to set env vars from test code :-/ it creates unnecessary dependency

Why do we need it? Why can't address client be not fake here?

Reply:
:-)